### PR TITLE
DOC: Update top links in landing page

### PR DESCRIPTION
**Pull request by bjnath**
Same intent as PR #17365 -- point "Installation" links to Hugo, start
page with "What is NumPy", let Hugo page link to "Troubleshooting ImportError".

### CODE REVIEW

File: doc/source/user/troubleshooting-importerror.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+:orphan:
```
Comment (eric-wieser) [inline doc/source/user/troubleshooting-importerror.rst:1]: Can you add an rst comment here explaining why this is an orphan? One reason answer would be "the main webpage links here" or "the url to this page appears in an error message.

Comment (bjnath) [reply doc/source/user/troubleshooting-importerror.rst:1]: Good suggestion. In 50811dc.

**Comment by melissawm** [comment]
This looks good to me, thanks @bjnath . The only detail that bugs me is that the installation link in the main menu does not have a subheader and looks different from all the others. But I don't think there's anything we could write there that would be clearer than the title.

**Review by melissawm — APPROVED** [review]

**Comment by charris** [comment]
Thanks Ben.

**Pull request merged**